apache/superset
# fix: Build scripts

### SUMMARY
Fixes broken links and redirections to make the build scripts succeed.

### AFTER

<img width="1218" alt="Screen Shot 2022-01-27 at 15 39 28" src="https://user-images.githubusercontent.com/60598000/151380778-91ba6698-819c-4f38-b076-cdb9ed70e277.png">

### TESTING INSTRUCTIONS
1. Launch `npm run build`
2. Make sure it succeeds

### ADDITIONAL INFORMATION
